helm/alfresco-content-services/tests/deployment-repository_test.yaml

```diff
@@ -95,3 +95,72 @@ tests:
           path: spec.template.spec.containers[0].envFrom[2].secretRef.name
           value: whatever
         template: deployment-repository.yaml
+
+  - it: should have configmap with CATALINA_OPTS with encryption.ssl.truststore.location set
+    values: *testvalues
+    asserts:
+      - matchRegex:
+          path: data.CATALINA_OPTS
+          pattern: |-
+            (^|\w\ )-Dencryption.ssl.truststore.location=\$JAVA_HOME/lib/security/cacerts($|\ )
+        template: config-repository.yaml
```

Comment:
there is a dedicated unit test file I recently created specific for config-repository, move those as well

you will need to update the regexp prefix that I remember to have updated recently, you will remember the discussion we had